index.d.ts

```diff
@@ -0,0 +1,13 @@
+declare module "svelte-virtual-list" {
+  import { SvelteComponentTyped } from "svelte/types/runtime";
+
+  export interface IProps<T> {
```

Comment:
Just a personal preference, but I suggest to name the interface `VirtualListProps` instead. Makes it more visible where it's coming from and less prone to name clashes.

Reply:
Sure. will do so.